plugin.program.newgrounds/addon.py

```diff
@@ -0,0 +1,198 @@
+import urllib,urllib2,urlparse
```

Comment:
New add-ons or add-on updates must support python3. Kodi 19 will support only python 3. Hence any add-on submitted to Leia must support both versions

Reply:
I'm confused about this one. What can I do to make sure it supports Python 3?

Reply:
urllib2 does not exist in python3. Your best option is probably to use script.module.six. Kodi alpha versions already use python3 so, install it and check :)